components/contact/ContactForm.tsx

```diff
@@ -24,10 +24,11 @@ export default function ContactForm() {
 
     emailjs
       .sendForm(
-        process.env.SERVICE_ID,
-        process.env.TEMPLATE_ID,
+        "service_l76u8u8",
+        "template_msyezf2",
         form.current,
```

Comment:
```suggestion
        form.current!,
```

Tells typescript this won't be a null value